controls/SV-230379.rb

```diff
@@ -4,51 +4,37 @@
 for individuals not requiring access to the system and application accounts for
 applications not installed on the system.'
-  desc 'check', 'Verify all accounts on the system are assigned to an active system,
-application, or user account.
+  desc 'check', 'Verify that there are no unauthorized interactive user accounts with the following command:
 
-    Obtain the list of authorized system accounts from the Information System
-Security Officer (ISSO).
+$ less /etc/passwd
 
-    Check the system accounts on the system with the following command:
+root:x:0:0:root:/root:/bin/bash
+...
+games:x:12:100:games:/usr/games:/sbin/nologin
+scsaustin:x:1001:1001:scsaustin:/home/scsaustin:/bin/bash
+djohnson:x:1002:1002:djohnson:/home/djohnson:/bin/bash
 
-    $ sudo more /etc/passwd
+Interactive user account, generally will have a user identifier (UID) of 1000 or greater, a home directory in a specific partition, and an interactive shell.
 
-    root:x:0:0:root:/root:/bin/bash
-    bin:x:1:1:bin:/bin:/sbin/nologin
-    daemon:x:2:2:daemon:/sbin:/sbin/nologin
-    sync:x:5:0:sync:/sbin:/bin/sync
-    shutdown:x:6:0:shutdown:/sbin:/sbin/shutdown
-    halt:x:7:0:halt:/sbin:/sbin/halt
-    games:x:12:100:games:/usr/games:/sbin/nologin
-    gopher:x:13:30:gopher:/var/gopher:/sbin/nologin
+Obtain the list of interactive user accounts authorized to be on the system from the system administrator or information system security officer (ISSO) and compare it to the list of local interactive user accounts on the system.
 
-    Accounts such as "games" and "gopher" are not authorized accounts as
-they do not support authorized system functions.
+If there are unauthorized local user accounts on the system, this is a finding.'
+  desc 'fix', 'Remove unauthorized local interactive user accounts with the following command where <unauthorized_user> is the unauthorized account:
 
-    If the accounts on the system do not match the provided documentation, or
-accounts that do not support an authorized system function are present, this is
-a finding.'
-  desc 'fix', 'Configure the system so all accounts on the system are assigned to an
-active system, application, or user account.
-
-    Remove accounts that do not support approved system activities or that
-allow for a normal user to perform administrative-level actions.
-
-    Document all authorized accounts on the system.'
+$ sudo userdel <unauthorized_user>'
   impact 0.5
-  ref 'DPMS Target Red Hat Enterprise Linux 8'
   tag severity: 'medium'
   tag gtitle: 'SRG-OS-000480-GPOS-00227'
   tag gid: 'V-230379'
-  tag rid: 'SV-230379r627750_rule'
+  tag rid: 'SV-230379r1017190_rule'
   tag stig_id: 'RHEL-08-020320'
-  tag fix_id: 'F-33023r567884_fix'
+  tag fix_id: 'F-33023r1014800_fix'
   tag cci: ['CCI-000366']
   tag nist: ['CM-6 b']
   tag 'host'
   tag 'container'
 
+  # Stores a list of user accounts that are not in the combined list of known system and user accounts
```

Comment:
```suggestion
  # Identifies a list of user accounts that are not in the combined list of known system and user accounts
```

"Stores" sounds to me like it is a predetermined input from the user, but really this is setting failing users to any user not in the predefined lists, correct?

Based on the STIG revision history: "RHEL-08-020320 - Updated Check and Fix texts."
TODO: Update the comment for clarity.

Reply:
Yes this is true. I agree with the language change.